bluebottle/activities/documents.py

```diff
@@ -114,11 +114,9 @@ def prepare_contribution_count(self, instance):
         return len(instance.contributions.filter(status__in=('new', 'success')))
 
     def prepare_country(self, instance):
-        if hasattr(instance, 'location') and instance.location:
-            return instance.location.country_id
```

Comment:
An activity might have a geolocation. Are you sure we only want to filter on initiative location?

Reply:
Ah you do so in specific activity :-) I see now